tests/timeseries/test_logical_meter.py

```diff
@@ -68,7 +68,7 @@ async def _synchronize_receivers(
         receivers: list[FormulaReceiver | FormulaReceiver3Phase | Receiver[Sample]],
     ) -> None:
         by_ts: dict[
-            datetime, List[FormulaReceiver | FormulaReceiver3Phase | Receiver[Sample]]
+            datetime, list[FormulaReceiver | FormulaReceiver3Phase | Receiver[Sample]]
         ] = {}
         for recv in receivers:
```

Comment:
This commit should probably be squash with previous one.
Otherwise tests don't pass.

Reply:
Because we have merge commits now,  this is technically not a problem anymore.  And I've been trying to not follow that rule too strictly,  especially when commits are too long already,  or when it helps with sequential readability.

The previous 3-phase PR also has this problem for example,  with two large commits,  and I think I've done this in other PRs also.

The main reason we were doing this was so that we can use `git bisect` to identify when issues were introduced.  And when we were using rebase merges earlier,  we needed all commits to pass tests,  else it wasn't possible to bisect.

But now,  we have merge commits,  and we can bisect just the merge commits.  So we might as well prioritize readability in individual commits.

Reply:
(assuming I understand your comment correctly. please correct me if you're talking about something else)

Reply:
Ok, I didn't know. Thanks.